### PR TITLE
Optionally show hidden sort options on all pages (random, Q&A, etc.)

### PR DESCRIPTION
**Pull request by erikdesjardins**
Fixes an issue [from reddit.](https://www.reddit.com/r/Enhancement/comments/33ocbr/feature_request_add_random_to_the_comments)

### CODE REVIEW

**Comment by andytuba** [comment]
heck, show em all?

**Comment by erikdesjardins** [comment]
Sure, sounds good.

File: lib/modules/betteReddit.js [hunk 1]
```diff
@@ -121,6 +121,12 @@ addModule('betteReddit', function(module, moduleID) {
 			type: 'boolean',
 			value: false,
 			description: 'Modify reddit\'s comment/link buttons ("perma-link source save...") such that they don\'t show up in searches. Disabled by default due to a slight performance impact.'
+		},
+		showHiddenSortOptions: {
+			type: 'boolean',
+			value: true,
```
Comment (erikdesjardins) [inline lib/modules/betteReddit.js]: Not sure if this should be enabled by default. I'm imagining a bunch of reports of "q&a sort is showing up on non-q&a pages".

Comment (andytuba) [inline lib/modules/betteReddit.js]: Yeah, false by default is a good idea.  I could envision other special-case sorts appearing in the future, too.

Comment (andytuba) [inline lib/modules/betteReddit.js]: Yeah, false by default is a good idea.  I could envision other special-case sorts appearing in the future, too.

**Pull request merged**